templates/deb/deb.changes.erb

```diff
@@ -14,7 +14,7 @@ Description: <%= firstline %>
 <%= remainder.collect { |l| l =~ /^ *$/ ? " ." : " #{l}" }.join("\n") %>
 <% end -%>
 Changes:
- <%= name %> (<%= "#{epoch}:" if epoch %><%= version %><%= "-" + iteration.to_s if iteration %>) whatever; urgency=medium
+<%= name %> (<%= "#{epoch}:" if epoch %><%= version %><%= "-" + iteration.to_s if iteration %>) <%= distribution %>; urgency=medium
```

Comment:
I think the leading whitespace removal will cause a breakage. Debian's changes file docs say this, from https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-changes - (Bold emphasis added by me)

> The first line of the field value (the part on the same line as Changes:) is always empty. The content of the field is expressed as **continuation lines, with each line indented by at least one space**. Blank lines must be represented by a line consisting only of a space and a full stop (.).

Reply:
Sorry, i didnt see the missing space. It sould be ok now.